gwt-editor-processor/src/test/resources/org/gwtproject/editor/processor/test08/result/TestEditor08_SimpleBeanEditorDelegate.java

```diff
@@ -0,0 +1,75 @@
+package org.gwtproject.editor.processor.test08;
+import java.lang.Object;
+import java.lang.Override;
+import org.gwtproject.editor.client.Editor;
+import org.gwtproject.editor.client.EditorVisitor;
+import org.gwtproject.editor.client.impl.SimpleBeanEditorDelegate;
+import org.gwtproject.editor.client.testing.FakeLeafValueEditorWithHasEditorDelegate_Long_SimpleBeanEditorDelegate;
+import org.gwtproject.editor.client.testing.FakeLeafValueEditorWithHasEditorDelegate_String_SimpleBeanEditorDelegate;
+import org.gwtproject.editor.processor.common.Model01Dto;
+public class TestEditor08_SimpleBeanEditorDelegate extends SimpleBeanEditorDelegate {
```

Comment:
looks like we're still missing generics here

Reply:
correct ... that is another upcoming PR